packages/browser/src/tracekit.ts

```diff
@@ -911,6 +912,21 @@ TraceKit._computeStackTrace = (function _computeStackTraceWrapper() {
     };
   }
 
+  function popFrames(stacktrace: any, popSize: number): any {
+    if (Number.isNaN(popSize)) {
```

Comment:
`Number.isNaN` is totally different than `isNaN`, since `Number.isNaN` detects if it's `NaN` or not, `isNaN` detects if it is `Not A Number`, you should also check value `0` as your previous logic: `if (isNaN(popSize) || popSize === 0)`

```js
Number.isNaN('sentry'); // false
isNaN('sentry'); // true
```

BTW, `Number.isNaN` seems not work so good on IE browsers, haven't checked yet. Would be nice if I can get some information abt it from yours XD

Reply:
@jiananshi I know how they work, thanks for pointing that out anyway :)
I can skip `0` logic, as `slice(0)` returns the same array, thus passing `popSize === 0` will be effectively no-op.

Yes, `Number.isNaN` is not IE friendly, but we already use it in our codebase. It's mentioned here https://docs.sentry.io/platforms/javascript/#browser-table

Reply:
Fine, let's take a step back and compare the previous code:

```js
// previous
if (typeof popSize !== 'number' || popSize === 0)

// current
if (Number.isNaN(popSize))
```

In the previous logic you do want a **NUMBER** but currently anythings besides `NaN` is acceptable, not sure if that's what you want.

btw the browser-table is a nice design, just learned that.

Reply:
@jiananshi

![image](https://user-images.githubusercontent.com/1523305/63383598-626b6d00-c39d-11e9-8405-9a171c9e8922.png)

Thankfully it will work just fine anyway, as `slice` can accept anything and it'll disregard anything other than a number. Thus this check is completely redundant anyway. Will remove it in the next patch release. Thanks and excuse my stubbornness 😅

Reply:
It's ok, as english is not my mother language it may feel little bit annoying, if it is, pls don't mind.

Reply:
It's totally fine, my bad :)

Reply:
> Will remove it in the next patch release. 

Removed (will be commited soon) :)